bothub/common/models.py

```diff
@@ -966,6 +966,10 @@ def clone_self(self, new_owner_id, language) -> Tuple[bool, str, int]:
         group_tasks()
         return repository_clone.pk, "Queued for cloning", status.HTTP_200_OK
 
+    def increment_trainings(self):
```

Comment:
Creating a method responsible for incrementing the `count_repository_train ` was a great choice. Encapsulates the field and centralizes the business rule, allowing reuse of code when necessary👏